azurerm/internal/services/network/tests/vpn_gateway_resource_test.go

```diff
@@ -224,6 +224,14 @@ resource "azurerm_vpn_gateway" "test" {
   bgp_settings {
     asn         = 65515
     peer_weight = 0
+
+    instance_bgp_peering_address {
```

Comment:
This isn't exactly clear, maybe it would be best to create two named blocks?
```suggestion
    instance_0_bgp_peering_address {
```